src/client.ts

```diff
@@ -30,6 +30,7 @@ type OnUserUpdateHandler = (userInfo: UserContext) => void
 export type OnReadyResult = {
   success: boolean
   reason?: string
+  dataReadyPromise?: Promise<any>
```

Comment:
Should we update the docs for `onReady` to mention that this will be in there, if it timed out?